### ISSUE TITLE
Some playlist appear empty

### ISSUE THREAD

**Issue by tcurdt** [pimusicbox/pimusicbox#347]
but they are not on Spotify.

**Comment by kingosticks** [comment]
Can you please describe more, I'm sorry but I don't understand what you mean.

**Comment by tcurdt** [comment]
I am not sure if this related to the web UI - but it shows some of my Spotify playlist as empty.

**Comment by kingosticks** [comment]
Right, I see, that is strange. Can you try them in an mpd client i.e.
`mpc playlist '<name of your playlist here>'`

**Comment by tcurdt** [comment]
OK. Tried. Other mpd clients work just fine.

**Comment by tcurdt** [comment]
Spoke to soon. Some work other mpd clients.
